app/views/pages/home.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="fluid-container text-center cover">
+  <div class="cover-inner">
+    <img src="" />
```

Comment:
We'll possibly change this to a background image when we receive some more assets, but we can cross that bridge when we come to it.